src/actions/catActions.js

```diff
@@ -0,0 +1,34 @@
+
+import * as types from './actionTypes';
+import axios from 'axios';
+
+export const instance = axios.create({
```

Comment:
rename this to something else other than "instance" we're using "instance" for our other axios objects. Maybe name it to "giphy_instance"